src/Tests/SenseNet.Services.Tests/PortalAuthenticationModuleTests.cs

```diff
@@ -334,103 +334,103 @@ public void OnAuthenticateRequestTokenRefreshWithInvalidTokenTest()
-            mockRequest.SetupGet(o => o.IsSecureConnection).Returns(true);
-            var headers = new NameValueCollection();
-            headers.Add("X-Authentication-Type", "Token");
-            headers.Add("X-Access-Data", "yJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJpc3N1ZXIiLCJzdWIiOiJzdWJqZWN0IiwiYXVkIjoiYXVkaWVuY2UiLCJleHAiOjIwOTA1Nzc1MDEsImlhdCI6MTQ5MDU3NzUwMSwibmJmIjoxNDkwNTc3NTAxLCJuYW1lIjoiTXlOYW1lIiwicm9sZSI6Ik15Um9sZSJ9");
-            mockRequest.SetupGet(o => o.Headers).Returns(headers);
-            var inCookies = new HttpCookieCollection();
-            inCookies.Add(new HttpCookie("as", "u1ESlK6iUZmX8T5a_AB4U74vtH4x2GGoQ-rNdN-JG6UTQQFsfFrhSX6bIq1S_pFq-Qd4Y__VRMJC31hHbdie6g")); ;
-            mockRequest.SetupGet(o => o.Cookies).Returns(inCookies);
-            var mockResponse = new Mock<HttpResponseBase>();
-            var cookies = new HttpCookieCollection();
-            string body = "";
-            int responseStatus = 0;
-            mockResponse.SetupGet(o => o.Cookies).Returns(cookies);
-            mockResponse.Setup(o => o.Write(It.IsAny<string>())).Callback((string t) => { body = t; });
-            mockResponse.SetupSet(o => o.StatusCode = It.IsAny<int>()).Callback((int s) => { responseStatus = s; });
-
-            var mockContext = new Mock<HttpContextBase>();
-            mockContext.SetupGet(o => o.Request).Returns(mockRequest.Object);
-            mockContext.SetupGet(o => o.Response).Returns(mockResponse.Object);
-            IPrincipal user = null;
-            mockContext.SetupSet(o => o.User = It.IsAny<IPrincipal>()).Callback((IPrincipal principal) => { user = principal; });
-            Configuration.TokenAuthentication.Audience = "audience";
-            Configuration.TokenAuthentication.Issuer = "issuer";
-            Configuration.TokenAuthentication.Subject = "subject";
-            Configuration.TokenAuthentication.EncriptionAlgorithm = "HS512";
-            Configuration.TokenAuthentication.AccessLifeTimeInMinutes = 10000000;
-            Configuration.TokenAuthentication.RefreshLifeTimeInMinutes = 0;
-            Configuration.TokenAuthentication.ClockSkewInMinutes = 1;
-            Configuration.TokenAuthentication.SymmetricKeySecret = "very secrety secret";
-            var application = new HttpApplication();
-            var module = new Mock<PortalAuthenticationModule>();
-            InitDependecies(module, mockContext, mockRequest, mockResponse);
-
-            module.Object.OnAuthenticateRequest(application, EventArgs.Empty);
-
-            Assert.Null(mockResponse.Object.Cookies["as"]);
-            Assert.Null(mockResponse.Object.Cookies["rs"]);
-            Assert.DoesNotContain("\"access\":", body);
-            Assert.DoesNotContain("\"refresh\":", body);
-            Assert.True(user is ClaimsPrincipal);
-            Assert.Equal("Visitor", user.Identity.Name);
-            Assert.Equal(0, responseStatus);
-        }
+        //[Fact]
```

Comment:
If there is no replacement for this test and it is obsolete or incorrect in the current form, we should delete it.